src/huggingface_hub/inference/_client.py

```diff
@@ -1224,12 +1226,11 @@ def image_to_image(
         image: ContentT,
         prompt: Optional[str] = None,
         *,
-        negative_prompt: Optional[str] = None,
-        height: Optional[int] = None,
-        width: Optional[int] = None,
+        negative_prompt: Optional[List[str]] = None,
         num_inference_steps: Optional[int] = None,
         guidance_scale: Optional[float] = None,
         model: Optional[str] = None,
+        target_size: Optional[ImageToImageTargetSize] = None,
```

Comment:
(nit) worth mentioning this breaking change in the release notes

Reply:
btw, I did not add a proper deprecation given the possible limited usage of `InferenceClient.image_to_image()`, but of course, this will be mentioned in the release notes!

Reply:
yep, no need for a deprecation cycle here :+1: